### PR TITLE
Fix/annotations

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by tieswesterborg** [comment]
Code looks good. Please test if everything still works. Ill test too

File: src/main/java/application/controllers/GraphController.java [hunk 1]
```diff
@@ -239,6 +239,9 @@ public void focus(int nextLevelNodeId) {
             prevClick.focus();
         }
         sideFocus(true);
+//        slideToPercent((prevClick.getLayoutX() - (mainController.getScreen().getWidth() / 4))
+//                / (graph.getMaxWidth() - 450));
+
```
Comment (ric-jon) [inline src/main/java/application/controllers/GraphController.java:244]: Commented code should be removed

File: src/main/java/application/controllers/GraphController.java [hunk 2]
```diff
@@ -239,6 +239,9 @@ public void focus(int nextLevelNodeId) {
             prevClick.focus();
         }
         sideFocus(true);
+//        slideToPercent((prevClick.getLayoutX() - (mainController.getScreen().getWidth() / 4))
```
Comment (tieswesterborg) [inline src/main/java/application/controllers/GraphController.java:242]: Were do we scroll to annotation now?

File: src/main/java/application/controllers/MainController.java [hunk 3]
```diff
@@ -529,40 +531,43 @@ private void setAnnotationButtonsActionListener(Button highlightButton, Button d
...
-    public void initListenerProperties() {
+    public void initListenerProperties(String input) {
         List<Annotation> annotations = graphController.getGraph().getModel().getAnnotations();
+        Map<Integer, Cell> cellMap = graphController.getGraph().getModel().getCellMap();
+
         try {
-            Annotation newAnn = AnnotationProcessor.findAnnotation(annotations,
-                    annotationTextField.getText());
-            Map<Integer, Cell> cellMap = graphController.getGraph().getModel().getCellMap();
+            System.out.println(input);
```
Comment (arbreurkes) [inline src/main/java/application/controllers/MainController.java:543]: System.out... should be removed

File: src/test/java/core/annotation/AnnotationProcessorTest.java [hunk 4]
```diff
@@ -94,6 +87,10 @@ public void testFindAnnotation() throws AnnotationProcessor.TooManyAnnotationsFo
         annotationList.add(a1);
         annotationList.add(a2);
 
+        for (Annotation ann : annotationList) {
+            System.out.println(ann.getDisplayNameAttr());
```
Comment (arbreurkes) [inline src/test/java/core/annotation/AnnotationProcessorTest.java:91]: System.out......

File: src/test/java/core/graph/GraphReducerTest.java [hunk 5]
```diff
@@ -99,107 +96,107 @@ public void testDetermineParentsDoubleParent() {
...
+//                assertEquals(1, nodeMap.get(4).getCollapseLevel());
+//
+//                assertNull(nodeMap.get(3));
+//
+//                if (i == 4) {
+//                    assertNotEquals(CellType.BUBBLE, nodeMap.get(4));
+//                }
+//            }
+//        }
+//    }
```
Comment (arbreurkes) [inline src/test/java/core/graph/GraphReducerTest.java:199]: 100 lines of commented code???

File: src/test/java/core/graph/GraphReducerTest.java [hunk 6]
```diff
@@ -272,58 +269,58 @@ public void testGenomesInIndel2() {
...
+//        nodeMap.get(6).setLinks(new ArrayList<>(Arrays.asList(7)));
+//
+//        GraphReducer.setStartMapSize(nodeMap.size());
+//        nodeMap = GraphReducer.collapse(nodeMap);
+//        assertEquals(1, nodeMap.get(1).getCollapseLevel());
+//        assertEquals(2, nodeMap.get(2).getCollapseLevel());
+//        assertEquals(1, nodeMap.get(4).getCollapseLevel());
+//        assertEquals(2, nodeMap.get(5).getCollapseLevel());
+//        assertEquals(1, nodeMap.get(7).getCollapseLevel());
+//    }
```
Comment (arbreurkes) [inline src/test/java/core/graph/GraphReducerTest.java:323]: another lot of commented code?

**Comment by tieswesterborg** [comment]
Not good.

**Pull request closed without merging**